### PR TITLE
Use correct colors in split panes

### PR DESCRIPTION
**Pull request by Flaque**
# Changes made
* Change split pane color to be from the theme rather than hard coded
* Added "color" for easy color manipulation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,20 +8,13 @@
...
-      "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_small_dataset.json",
-    "add-many-notes":
-      "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_large_dataset.json",
+    "db-down": "node node_modules/db-migrate/bin/db-migrate down --config config/database.json",
+    "db-up": "node node_modules/db-migrate/bin/db-migrate up --config config/database.json",
+    "db-reset": "node node_modules/db-migrate/bin/db-migrate reset --config config/database.json",
+    "db-create-config": "cross-env NODE_ENV=dev babel-node scripts/create-db-config/create.js",
+    "db-migrate-create": "node node_modules/db-migrate/bin/db-migrate create -e dev --config config/database.json",
+    "add-some-notes": "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_small_dataset.json",
+    "add-many-notes": "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_large_dataset.json",
```
Comment (kmccrohan) [inline package.json]: not a big issue, but I've noticed our different linters seems to keep changing the formatting of these longer scripts, so we should probably try to figure out how to make it consistent.

Comment (Flaque) [reply package.json]: I believe it's a fight between prettier and npm. Prettier will try and format the package.json and then npm (whenever we install something) will remake the file in an unformatted fashion. 

We could probably get visual studio and prettier to ignore formatting this file. 

Or we could make the scripts smaller?

Comment (kmccrohan) [reply package.json]: Let's not try to make the scripts smaller... we've wasted enough time on that already. I've already added this file to files my prettier ignores so it stops trying to format it.

Comment (Flaque) [reply package.json]: ?? Wait where's your prettier ignores? Can you add that to the project?

Comment (kmccrohan) [reply package.json]: It was in the settings of atom that I changed it. not sure how to do that for everyone.

**Pull request merged**